HealthcareSystem/PatientWebApp/Controllers/SurveyController.cs

```diff
@@ -33,7 +36,7 @@ public SurveyController(ISurveyService surveyService, IExaminationService examin
             _examinationValidator = new ExaminationValidator(_examinationService);
         }
 
-
+        [Authorize(Roles = UserRoles.Patient)]
         [HttpPost]
         public ActionResult AddSurvey(SurveyDTO surveyDTO)
         {
```

Comment:
Patients can only fill out the survey for their own examinations, that needs to be checked as well.